peps/pep-9999.rst

```diff
@@ -0,0 +1,113 @@
+PEP: 9999
+Title: imath --- module for number-theoretic functions
+Author: Sergey B Kirpichev
+Sponsor: TBD
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 10-May-2025
+Python-Version: 3.15
+Post-History: 02-Jun-2019,
+              `09-May-2025 <https://discuss.python.org/t/91337>`__,
+
+
+Abstract
+========
+
+This PEP describes new module for number-theoretic functions over integers,
```

Comment:
Just a few friendly suggestions.  Take them if you want 😄 

By definition, "number-theoretic" implies over integers.  So you could just say "a new module for number-theoretic functions.   This includes modular arithmetic, integer roots, primality, factorization, integer sequences, combinatorics,  bitwise utilities, and number encodings."